regression-test/suites/db_sync/prop_incsync/auto_increment/test_ds_prop_auto_increment.groovy

```diff
@@ -0,0 +1,105 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("test_ds_prop_incsync_auto_increment") {
+    def helper = new GroovyShell(new Binding(['suite': delegate]))
+            .evaluate(new File("${context.config.suitePath}/../common", "helper.groovy"))
+
+    def dbName = context.dbName
+    def tableNameFull = "tbl_full"
+    def tableNameIncrement = "tbl_incr"
+    def test_num = 0
+    def insert_num = 5
+
+    def exist = { res -> Boolean
+        return res.size() != 0
+    }
+
+    sql "DROP TABLE IF EXISTS ${dbName}.${tableNameFull}"
+    target_sql "DROP TABLE IF EXISTS TEST_${dbName}.${tableNameFull}"
+    sql "DROP TABLE IF EXISTS ${dbName}.${tableNameIncrement}"
+    target_sql "DROP TABLE IF EXISTS TEST_${dbName}.${tableNameIncrement}"
+
+
+    helper.enableDbBinlog()
+    helper.ccrJobDelete()
+    helper.ccrJobCreate()
+
+    sql """
+            CREATE TABLE ${tableNameFull} (
+                `id` BIGINT NOT NULL AUTO_INCREMENT,
+                `value` int(11) NOT NULL
+            ) ENGINE=OLAP
+            DUPLICATE KEY(`id`)
+            PROPERTIES (
+            "replication_allocation" = "tag.location.default: 1"
+            )
+    """
+
+    for (int index = 0; index < insert_num; index++) {
+        sql "INSERT INTO ${tableNameFull} (value) VALUES (${insert_num})"
+    }
+    sql "sync"
+
+    assertTrue(helper.checkRestoreFinishTimesOf("${tableNameFull}", 30))
+
+    assertTrue(helper.checkShowTimesOf("SHOW TABLES LIKE \"${tableNameFull}\"", exist, 60, "sql"))
+
+    assertTrue(helper.checkShowTimesOf("SHOW TABLES LIKE \"${tableNameFull}\"", exist, 60, "target"))
+
+    def target_res = target_sql "SHOW CREATE TABLE ${tableNameFull}"
+
+    assertTrue(target_res[0][1].contains("`id` bigint NOT NULL AUTO_INCREMENT(1)"))
+
+    res = sql "select * from ${tableNameFull} order by id"
+
+    target_res = target_sql "select * from ${tableNameFull} order by id"
+
+    assertEquals(target_res, res)
+
+    sql """
+            CREATE TABLE ${tableNameIncrement} (
+                `id` BIGINT NOT NULL AUTO_INCREMENT,
```

Comment:
`id` BIGINT NOT NULL AUTO_INCREMENT(100),
We should cover setting intial value for auto increment.